providers/zohocrm/parse.go

```diff
@@ -0,0 +1,47 @@
+package zohocrm
+
+import (
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/jsonquery"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+	"github.com/spyzhov/ajson"
+)
+
+/*
+   "data": {...},
```

Comment:
is this object or a list? This is contradicting from what I saw when you handle 304 status code. Lets have API link here as reference.

Reply:
A list, Thanks.